plugin/src/test/java/org/opensearch/ml/rest/RestMLUpdateConnectorActionTests.java

```diff
@@ -114,8 +114,15 @@ public void testUpdateConnectorRequest() throws Exception {
         assertEquals("2", updateConnectorRequest.getUpdateContent().getVersion());
     }
 
+    public void testUpdateConnectorRequestWithParsingException() throws Exception {
+        exceptionRule.expect(OpenSearchParseException.class);
+        exceptionRule.expectMessage("Can't get text on a VALUE_NULL");
```

Comment:
Just curious where are we setting this message? Can we have more readable message from customer's point of view?

Reply:
This is the message from the XContentParser in OpenSearch, which contains the exact line and coordinate where the null comes from. exceptionRule.expectMessage only checks if the real message contains this "Can't get text on a VALUE_NULL". The means the real message returned to customer is more explanatory.

Reply:
Thank you for the clarification? Just out of curiosity, do we know what's the final message we send to customer?

Reply:
Yes, please check the description of the PR which has the exact coordinate of the error. Also please note this is one example of the paring exception.